generator/template.go

```diff
@@ -456,6 +456,11 @@ func GetTemplate(r *registry.Registry) *template.Template {
 
 func fieldName(r *registry.Registry) func(name string) string {
 	return func(name string) string {
+		// if explicit mapping exists, use it
+		val, ok := r.ExplicitCamelCaseMapping[name]
```

Comment:
That will kill 2 birds with 1 stone.
1. First issue that is going to be fixed is the bad prop rename where e.g. `a1c` or `t2d` is translated to `a1C` and `t2D`. Known bug: https://github.com/grpc-ecosystem/protoc-gen-grpc-gateway-ts/issues/52
2. The issue Michelle had a couple of months ago. See first bullet point here: https://cadencerpm.atlassian.net/browse/PLAT-4159